encode.go

```diff
@@ -0,0 +1,22 @@
+package goxy
+
+import (
+	"bytes"
+	"encoding/gob"
+)
+
+func Encode(m interface{}) (string, error) {
+	target := &bytes.Buffer{}
+	err := gob.NewEncoder(target).Encode(m)
+	if err != nil {
+		return "", err
+	}
+	return target.String(), nil
```

Comment:
Why don't we just return the bytecode? why do we need to convert it to string?

Reply:
because the  redis package that we use in our projects uses string for GET 
and besides , to have bytes we should call .Bytes() on target and it's on way or another
but for the sake of easiness I used string.